src/audio_midi.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EP_AUDIO_MIDI_H
+#define EP_AUDIO_MIDI_H
+
+// Headers
+#include "audio_decoder.h"
+#include "midisequencer.h"
+
+#if defined(GEKKO) || defined(_3DS)
+#  define EP_MIDI_FREQ 22050
+#else
+#  define EP_MIDI_FREQ 44100
```

Comment:
I'd suggest 22050 also for other platforms. After comparing with scc1t2.sf2, they sound similar to the original GM.DLS DirectMusic playback, so it seems Windows plays at that rate from RPG_RT.exe and editor. DLS samples are 22 KHz according to Awave Studio for middle note playback, so there wouldn't be a major gain with higher playback rate.